rest-api/data_gen/fake_participant_generator.py

```diff
@@ -489,13 +489,17 @@ def _submit_hpo_changes(self, participant_response, participant_id, consent_time
 
   def _submit_status_changes(self, participant_response, participant_id, last_request_time):
```

Comment:
It looks like this is called immediately after the response is fetched; why does it need to be re-fetched?

Reply:
Chatted offline -- adding a comment will be useful, but makes sense now.

Reply:
Added comments.